shell/platform/windows/flutter_windows_win32.cc

```diff
@@ -69,10 +68,7 @@ bool FlutterDesktopViewControllerHandleTopLevelWindowProc(
 }
 
 uint64_t FlutterDesktopEngineProcessMessages(FlutterDesktopEngineRef engine) {
```

Comment:
I wonder if we even need FlutterDesktopEngineProcessMessages any more with this change.  I don't believe we use it in UWP.  It would be a breaking change tho.

Reply:
It's not necessary anymore (it a noop that just returns -1). Next step would be to remove this from the Windows template (it would become simple Get/Translate/Dispatch message loop). Returning -1 ensures that existing run-loops work just fine after the change.

Reply:
Actually removing the function would involve writing a migrator into the Flutter tool. (There's some infrastructure in the tooling for it, but it hasn't been used yet on Windows)

Reply:
@stuartmorgan, I don't think I'd want to remove the function from API anytime soon, but the template can be significantly simplified. I'll look at the migration, but that should not block this PR, right?

Reply:
Right, both this PR and a template upgrade are non-breaking, do aren't blocked. It's only at the point where it would actually be removed that a migrator would be necessary.